server/routerlicious/packages/services-telemetry/src/lumber.ts

```diff
@@ -114,7 +115,7 @@ export class Lumber<T extends string = LumberEventName> {
         message: string,
         logLevel: LogLevel,
         successful: boolean,
-        exception: Error | undefined) {
+        exception: any | undefined) {
```

Comment:
With this change in typing, ss there an advantage to moving the responsibility of wrapping the given exception into an Error object from lumberjack users to the framework? Seems like In the redis client code scenario (or others like it), the users could potentially have more context to better construct the error object than the framework, which can only stringify.

Reply:
I think it's a trade off between the ease of calling Lumberjack and just giving it anything vs. having the ability to create more contextualized Error objects, but at the expense of possibly duplicating some logic. @znewton had discussed this originally with me and I thought it would be better to put the centralized logic in Lumberjack. The user can still use the context to create a more contextualized Error object and then give it to Lumberjack, but we are also adding the flexibility of accepting other types of errors. What do you think?

Reply:
Makes sense, thanks!